apps/heureka/src/components/Services/Services.tsx

```diff
@@ -7,19 +7,21 @@ import React, { useState } from "react"
 import { Breadcrumb } from "../common/Breadcrumb"
 import { Filters } from "../common/Filters"
 import { ServicesList } from "./ServicesList"
-import { Panel } from "../common/Panel"
 import { FilterSettings } from "../common/Filters/types"
 import { useFetchServiceFilters } from "./useFetchServiceFilters"
 import { InitialFilters } from "../../App"
 import IssuesCount from "./ServicesList/IssuesCount"
-import { useFetchServices } from "./useFetchServices"
+import { useStore } from "../../store/StoreProvider"
 
 export type ServiceType = {
   id: string
   name: string
   issuesCount: {
```

Comment:
It would be good here to reuse the existing types so we have just 1 source of thruth:
https://github.com/cloudoperators/juno/blob/main/apps/heureka/src/components/Services/useFetchServicesCounts.ts#L15